engine/src/cmd/music.cpp

```diff
@@ -491,11 +533,12 @@ void Music::SkipRandSong( int whichlist, int layer )
 {
     if (muzak) {
         if (layer < 0) {
-            if (muzak_count >= 2)
+            if (muzak_count >= 2){
                 muzak[muzak_cross_index]._SkipRandSong( whichlist );
-
-            else
+            }
```

Comment:
In all other places you have `} else {` on the same line and here on a different. not a blocker, just noting.